packages/cli/commands/remove.js

```diff
@@ -40,7 +41,8 @@ exports.handler = async options => {
   trackCommandUsage('remove', {}, accountId);
 
   try {
-    await deleteFile(accountId, hsPath);
+    let unixPath = convertToUnixPath(hsPath);
```

Comment:
The expectation here is that the path the user provides will be in HubSpot format so can we remove this?